sdk/core/Azure.Core/tests/public/Internal/ModelReaderWriterHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Runtime.CompilerServices;
+
+namespace Azure.Core.Tests.Public
+{
+    internal static class ModelReaderWriterHelper
```

Comment:
Why duplicate this across test projects instead of share it through common like we do with the shared models?

Reply:
Trying to avoid introducing shared source.

Reply:
You have three copies of this file in your branch, though - that seems excessive.  Could one of the common test libraries make it public to be shared across the libraries, just like your public models are?

Reply:
This will all change once the shared source is solved so I don't want to do too much with it right now.  Do you think this should block beta?